### DISCUSSION TITLE
Spotify integration

### DISCUSSION THREAD

**Discussion by tomwendel** [Wendelpunkt/ToypadChallenge#9] [Ideas]
Hows about a spotify integration to bind specific figures with playlists to have a figure for each mood. Not sure how to use the multifigure feature. maybe mix up playlists or something?

**Comment by EldoranDev** [comment]
I'd imagine something like  a hardware version of [Festify](https://festify.rocks/) where people could place figures on the reader and songs or playlists are going to get played, the index of the tag could be used to keep the order of the songs/playlists.

Alternatively it could mix songs from playlists as per your suggestion, by randomly picking songs from each of the present playlists.

**Comment by yannickschrade** [comment]
I would like to take this #21

**Comment by tomwendel** [comment]
Christian found that obe yesterday: https://github.com/meltaxa/musicfig
Thats a Spotify / MP3 Player controlled by the Toypad in Raspberry
